### PR TITLE
Remove `BraintreePaymentFlow` dependency from `BraintreeThreeDSecure` module

### PR DESCRIPTION
**Pull request by scannillo**
### Summary of changes

- After dropping 3DS 1, our 3DS module no-longer utilizes a web-browser switch fallback (only native UI via Cardinal)
- After the changes made in #1013, we can remove the PaymentFlow dependency from the 3DS module entirely
- Update MIGRATION guide, to separate 3DS and Local Payment sections

### Checklist

- [x] Added a changelog entry

### Authors
@scannillo

### CODE REVIEW

**Review by sshropshire — APPROVED** [review]

**Comment by sshropshire** [comment]
I just noticed it's in draft 🤦 . I'll circle back.

**Comment by scannillo** [comment]
>  I just noticed it's in draft 🤦 . I'll circle back.

@sshropshire - all good! I just did DRAFT to start to kick of CI

**Review by KunJeongPark — APPROVED** [review]

File: Sources/BraintreeThreeDSecure/BTThreeDSecureRequest.swift [hunk 1]
```diff
@@ -70,254 +66,7 @@ import BraintreePaymentFlow
...
-
-extension BTThreeDSecureRequest: BTThreeDSecureRequestDelegate {
-
-    public func onLookupComplete(
-        _ request: BTThreeDSecureRequest,
-        lookupResult result: BTThreeDSecureResult,
-        next: (() -> Void)
-    ) {
-        next()
-    }
```
Comment (sshropshire) [inline Sources/BraintreeThreeDSecure/BTThreeDSecureRequest.swift:322]: 🔥

**Review by sshropshire — APPROVED** [review]
Double 👍 !

**Pull request merged**